docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md

```diff
@@ -1,17 +1,18 @@
 ---
 weight: 1
 sort: 1
-title: Getting started with vmanomaly
+title: vmanomaly integration
```

Comment:
please rename it as well here - https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide with folder (in that case you need to change links too) or/and Readme.md

Reply:
thanks, renamed folder, changed ref, updated title reference